src/style.css

```diff
@@ -0,0 +1,72 @@
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
```

Comment:
Header has 100% width by default and body padding is 0.
It is not necessary here.